### PR TITLE
Fix typos in tests/

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: tests/zfs-tests/tests/functional/trim/trim.kshlib [hunk 1]
```diff
@@ -18,7 +18,7 @@
 . $STF_SUITE/tests/functional/cli_root/zpool_trim/zpool_trim.kshlib
 
 #
-# Get the actual on disk disk for the provided file.
+# Get the actual on disk for the provided file.
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/trim/trim.kshlib]: Done!

File: tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh [hunk 2]
```diff
@@ -64,7 +64,7 @@ for user in "${no_users[@]}"; do
 	log_mustnot zfs set userquota@$user=100m $QFS
 done
 
-log_note "can set all numberic id even that id is not existed"
+log_note "can set all numeric id even that id is not existed"
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh]: Done!

File: tests/zfs-tests/tests/functional/userquota/userquota_010_pos.ksh [hunk 3]
```diff
@@ -33,7 +33,7 @@
 
 #
 # DESCRIPTION:
-#       Check userquota and groupquota be overwrited at same time
+#       Check userquota and groupquota be overwritten at same time
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/userquota/userquota_010_pos.ksh]: Done!

File: tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh [hunk 4]
```diff
@@ -37,7 +37,7 @@
 # should fail.
 #
 # STRATEGY:
-#	1. Create a file, and set an with an xattr
+#	1. Create a file, and set a with an xattr
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh]: Done!

File: tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh (hunk 2 above)
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh]: well, I guess I messed up with commits... Anyway, let's see now.

File: tests/zfs-tests/tests/functional/userquota/userquota_010_pos.ksh [hunk 5]
```diff
@@ -33,7 +33,7 @@
 
 #
 # DESCRIPTION:
-#       Check userquota and groupquota be overwrited at same time
+#       Check userquota and groupquota be overwritten at the same time
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/userquota/userquota_010_pos.ksh]: Done and squashed!

File: tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh [hunk 6]
```diff
@@ -64,7 +64,7 @@ for user in "${no_users[@]}"; do
 	log_mustnot zfs set userquota@$user=100m $QFS
 done
 
-log_note "can set all numberic id even that id is not existed"
+log_note "can set all numeric id even if that id is nonexistent"
```
Comment (rlaager) [inline tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh]: As I suggested in another PR, "can set all numeric id even if that id does not exist" might be more clear.

File: tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh [hunk 7]
```diff
@@ -37,8 +37,8 @@
 # should fail.
 #
 # STRATEGY:
-#	1. Create a file, and set an with an xattr
-#       2. Set the octal file permissions to 000 on the file.
+#	1. Create a file with an xattr
+#       2. Set the octal file permissions to 000 on the file
```
Comment (rlaager) [inline tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh]: I might simplify this to "2. Set the file permissions to 000"

File: tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh (hunk 6 above)
Comment (Gelma) [reply tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh]: Done!

File: tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh (hunk 7 above)
Comment (Gelma) [reply tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh]: Done!

File: tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh (hunk 2 above)
[the remark this answers is not in the document]
Comment (rlaager) [reply tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh]: You now have "even if that id does not exists". It needs to be "does not exist" (no "s").

File: tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh (hunk 7 above)
Comment (rlaager) [reply tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh]: I'm still seeing "2. Set the file permissions to 000 on the file". I was suggesting dropping "on the file".

File: tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh [hunk 8]
```diff
@@ -37,8 +37,8 @@
 # should fail.
 #
 # STRATEGY:
-#	1. Create a file, and set an with an xattr
-#       2. Set the octal file permissions to 000 on the file.
+#	1. Create a file with an xattr
+#       2. Set the file permissions to 000 on the file
```
Comment (freqlabs) [inline tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh]: While you're fixing up this commit, please align this properly.

Comment (Gelma) [reply tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh]: Done!

File: tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh [hunk 9]
```diff
@@ -64,7 +64,7 @@ for user in "${no_users[@]}"; do
 	log_mustnot zfs set userquota@$user=100m $QFS
 done
 
-log_note "can set all numberic id even that id is not existed"
+log_note "can set all numeric id even if that id does not exists"
```
Comment (freqlabs) [inline tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh]: Please change `exists` to `exist`.

File: tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh [hunk 10]
```diff
@@ -37,8 +37,8 @@
 # should fail.
 #
 # STRATEGY:
-#	1. Create a file, and set an with an xattr
-#       2. Set the octal file permissions to 000 on the file.
+#	1. Create a file with an xattr
+#   2. Set the file permissions to 000 on the file
```
[the remark this answers is not in the document]
Comment (Gelma) [reply tests/zfs-tests/tests/functional/xattr/xattr_003_neg.ksh]: Sorry, now should be right!

File: tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh (hunk 9 above)
Comment (Gelma) [reply tests/zfs-tests/tests/functional/userquota/userquota_005_neg.ksh]: Sorry! Now it should be right.

**Pull request merged**